packages/core/src/saml-applications/queries/configs.ts

```diff
@@ -16,7 +16,7 @@ export const createSamlApplicationConfigQueries = (pool: CommonQueryMethods) =>
   const updateSamlApplicationConfig = buildUpdateWhereWithPool(pool)(SamlApplicationConfigs, true);
 
   const findSamlApplicationConfigByApplicationId = async (applicationId: string) =>
-    pool.maybeOne<SamlApplicationConfig>(sql`
+    pool.one<SamlApplicationConfig>(sql`
```

Comment:
This config should be nullable?

Reply:
When creating a SAML app, a record in the `saml_application_configs` table is inserted simultaneously. Subsequent operations only involve DB "update" actions on this record.
If this record is not inserted during the creation of the SAML app, additional checks will be required when updates to the config are needed later. This means that we can always expect this query to retrieve exact ONE record.

Reply:
According to the design of the console, when creating a SAML app, you only need to provide `name`/`description`.

Reply:
name/description should be stored in the application table?